model/schema/highThroughputExpression.yaml

```diff
@@ -0,0 +1,577 @@
+    multivalued: false
+    required: false
+
+  sample_age: # sampleAge in agr_schemas json
+    description: >-
+      a collection of terms that when used together represent the age and stage of the sample
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: BioSampleAge
+
+  sample_age_dto: # sampleAge in agr_schemas json
+    description: >-
+      a collection of terms that when used together represent the age and stage of the sample
+    domain: HighThroughputExpressionDatasetSampleAnnotationDTO
+    range: BioSampleAge
+
+  sample_id:
+    description: >-
+      The IDs for this sample object, MOD or external database ID (ex: GEO); primaryId and secondaryId
+    domain: HighThroughputExpressionDatasetAnnotation
+    range: HighThroughputExpressionDatasetID
+    multivalued: false
+    required: false
+
+  sample_id_dto:
+    description: >-
+      The IDs for this sample object, MOD or external database ID (ex: GEO); primaryId and secondaryId
+    domain: HighThroughputExpressionDatasetAnnotationDTO
+    range: HighThroughputExpressionDatasetIDDTO
+    multivalued: false
+    required: false
+
+  sample_locations: # sampleLocations in agr_schemas json
+    description: >-
+      A collection of terms that when used together represents the location within an organism from which the sample was taken
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: AnatomicalSite
+    multivalued: true
+
+  sample_location_dtos: # sampleLocations in agr_schemas json
+    description: >-
+      A collection of terms that when used together represents the location within an organism from which the sample was taken
+    domain: HighThroughputExpressionDatasetSampleAnnotationDTO
+    range: AnatomicalSite
+    multivalued: true
+
+  sample_title: # sampleTitle in agr_schemas json
+    description: >-
+      The free text title of the sample
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: string
+
+  sample_type: # sampleType in agr_schemas json
+    description: >-
+      Ontology ID of the biosample type. EX: total mRNA, OBI:XXXXXXX. See enumeration below.
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: OBITerm
+    required: true
+
+  sample_type_curie: # sampleType in agr_schemas json
+    description: >-
+      Ontology ID of the biosample type. EX: total mRNA, OBI:XXXXXXX. See enumeration below.
+    domain: HighThroughputExpressionDatasetSampleAnnotationDTO
+    range: string
+    required: true
+
+  sequencing_format: # sequencingFormat in agr_schemas json
+    description: >-
+      single or paired end sequencing; might be better in experiment info
+    domain: HighThroughputExpressionDatasetSampleAnnotation
+    range: sequencing_format_values
+
+  stage:
+    description: >-
+      The life stage of an organism or biosample as expressed by a TemporalContext entity
+    domain: BioSampleAge
+    range: TemporalContext
+
+  sub_series:
```

Comment:
@chris-grove This really needs to be an association that is loaded after the datasets, otherwise we are reliant on the annotations being in the correct order when ingested.

Reply:
@markquintontulloch  I realized that the range of sub_series should just be list of IDs, as strings I suppose. The entities being referred to in the list are not represented in our database, only in GEO, ArrayExpress, or in some cases, a MOD. I will make the appropriate change to make this a list of strings.